test/extensions/filters/network/mysql_proxy/mysql_test_config.yaml

```diff
@@ -2,7 +2,7 @@ admin:
   access_log_path: /dev/null
```

Comment:
Question: Why are we using this config vs. just using the default config and config modifiers? Would that have made it easier to prevent this issue?

Reply:
I think we could have done it either way. However, @gvalentinivmw might know more. We could have still run into this issue as the config modifiers itself don't constrain IP addresses AFAIK.

Reply:
OK that's fine. It's not a big deal either way, though I think in general it's better to use the stock config in case that changes in the future.

Reply:
For the record, I think the problem here is related to a bug I reported a while back: https://github.com/envoyproxy/envoy/issues/5556

it'd be nice to make tests that do not care about ports not have to specify anything about them, or care about ipv6 vs ipv4.

Reply:
Yea, that'd definitely be helpful.